### ISSUE TITLE
question about backward model

### ISSUE THREAD

**Issue by Nuyoah003** [GameDisplayer/DRL4DG#14]
backward_encoder = EncoderRNN(hidden_size, embedding, encoder_n_layers, dropout)
backward_decoder = LuongAttnDecoderRNN(attn_model, embedding, hidden_size, voc.num_words, decoder_n_layers, dropout)

I have a question: why doesn't the backward model use a pre-trained seq2seq model for initialization? What is the purpose of redefining a backward model?
